### PR TITLE
updated parameters for trim and crop for create cropped video function

### PR DESCRIPTION
**Pull request by kcmcveigh**

### CODE REVIEW

**Review by GeorgeEfstathiadis — CHANGES_REQUESTED** [review]

File: openwillis/measures/video/util/crop_utils.py [hunk 1]
```diff
@@ -256,10 +256,10 @@ def create_cropped_frame(
         - 'bb_y' (int): The y-coordinate of the top-left corner of the bounding box.
         - 'bb_w' (int): The width of the bounding box.
         - 'bb_h' (int): The height of the bounding box.
-    darken : bool, optional
+    crop : bool, optional
         If True, blackens the area outside the bounding box. If False, crops the frame with padding and centering. Default is True.
     default_size : tuple, optional
-        The size of the cropped frame if `darken` is False. Default is (512, 512).
+        The size of the cropped frame if `crop` is False. Default is (512, 512).
```
[the remark this answers is not in the document]
Comment (kcmcveigh) [reply openwillis/measures/video/util/crop_utils.py]: completely right updated pr

File: openwillis/measures/video/util/crop_utils.py [hunk 2]
```diff
@@ -256,10 +256,10 @@ def create_cropped_frame(
         - 'bb_y' (int): The y-coordinate of the top-left corner of the bounding box.
         - 'bb_w' (int): The width of the bounding box.
         - 'bb_h' (int): The height of the bounding box.
-    darken : bool, optional
+    crop : bool, optional
         If True, blackens the area outside the bounding box. If False, crops the frame with padding and centering. Default is True.
```
[the remark this answers is not in the document]
Comment (kcmcveigh) [reply openwillis/measures/video/util/crop_utils.py]: completely right updated pr

File: openwillis/measures/video/util/crop_utils.py [hunk 3]
```diff
@@ -268,25 +268,25 @@ def create_cropped_frame(
 
     Notes:
     ------
-    - If `darken` is True, the function uses the `blacken_outside_bounding_box` function to blacken the area outside the bounding box.
-    - If `darken` is False, the function uses the `crop_with_padding_and_center` function to crop the frame with padding and centering.
+    - If `crop` is True, the function uses the `blacken_outside_bounding_box` function to blacken the area outside the bounding box.
+    - If `crop` is False, the function uses the `crop_with_padding_and_center` function to crop the frame with padding and centering.
```
[the remark this answers is not in the document]
Comment (kcmcveigh) [reply openwillis/measures/video/util/crop_utils.py]: completely right updated pr

File: openwillis/measures/video/util/crop_utils.py [hunk 4]
```diff
@@ -297,8 +297,8 @@ def create_face_frame(
         frame_dict (dict): A dictionary containing the coordinates of the face region in the frame.
         frame (numpy.ndarray): The input frame to be cropped.
         default_size (tuple, optional): The default size of the cropped face frame. Defaults to (512, 512).
-        keep_original_timing (bool, optional): Whether to keep the original timing of the frame. Defaults to True.
-        darken (bool, optional): Whether to darken the cropped face frame. Defaults to True.
+        trim (bool, optional): Whether to keep the original timing of the frame. Defaults to True.
```
[the remark this answers is not in the document]
Comment (kcmcveigh) [reply openwillis/measures/video/util/crop_utils.py]: updated line to clarify but you're correct

File: openwillis/measures/video/util/crop_utils.py [hunk 5]
```diff
@@ -385,7 +383,7 @@ def create_cropped_video(
     frame_width = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
     frame_height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
 
-    if darken == False:
+    if crop == True:
```
[the remark this answers is not in the document]
Comment (kcmcveigh) [reply openwillis/measures/video/util/crop_utils.py:386]: added in pr

**Review by GeorgeEfstathiadis — APPROVED** [review]

**Pull request merged**